test/contrib/test_pyopenssl.py

```diff
@@ -84,3 +87,14 @@ def test_get_subj_alt_name(self, mock_warning):
         self.assertEqual(mock_warning.call_count, 1)
         self.assertIsInstance(mock_warning.call_args[0][1],
                               x509.DuplicateExtension)
+
+class TestPyOpenSSLException(unittest.TestCase):
```

Comment:
Let's add this testcase to `TestClientCerts` to ensure it's run against all SSLContext implementations.